src/main/java/com/synopsys/integration/jenkins/detect/service/DetectArgumentService.java

```diff
@@ -103,7 +103,8 @@ private String escapeArgument(String argument, Function<String, String> escaper)
         String cleanedArg = argument;
         if (argument.startsWith("--") && argument.contains("=")) {
             String[] splitArgument = argument.split("=", 2);
-            cleanedArg = splitArgument[0] + "=" + escaper.apply(splitArgument[1]);
+            String endArg = splitArgument[1].endsWith("==") ? splitArgument[1] : escaper.apply(splitArgument[1]);
+            cleanedArg = splitArgument[0] + "=" + endArg;
```

Comment:
I'm confused by this. Any idea why this change would be required now, but not before? Also, is the endsWith("==") trying to detect base 64 encoded values that include "="? If so: I believe base64 encoded values can end with zero, one, or two "=" characters, so I'm wondering about the ends-with-one case... do we need to disable the escaper for that too? (Also, to me, this kind of thing seems to be asking for a unit test, especially if there really are 3 cases to handle.)

Reply:
Yes I had meant to improve that but dropped it on the floor.  I'll have a better way shortly.

Reply:
I made a change that does not rely on the "==".